upgrade/scripts/k8s/upgrade_k8s_job.yaml

```
@@ -44,7 +44,7 @@ spec:
           ssh -i /myssh/id_rsa -o StrictHostKeyChecking=no \
               -o UserKnownHostsFile=/dev/null root@ncn-m001 \
               "ts=\$(date +'%Y-%m-%d_%H-%M-%S'); \
```

If you don't set pipefail, then a failure of the `upgrade_k8s_steps.sh` script will be treated as a success, because it will just go by the exit code from the `tee` command (which will be 0, except in the case of an I/O error).
```suggestion
              "set -o pipefail; \
               ts=\$(date +'%Y-%m-%d_%H-%M-%S'); \
```

---

Updated sir.